### PR TITLE
Fix Deploy Image flake

### PR DESCRIPTION
**Pull request by andrewballantyne**
The test in #4187 had a lucky successful pass and pushed a highly flaky test - trying to fix it.

### CODE REVIEW

**Comment by andrewballantyne** [comment]
Looking at fixing the test instead of just disabling it...

File: frontend/integration-tests/tests/deploy-image.scenario.ts [hunk 1]
```diff
@@ -42,9 +42,12 @@ describe('Deploy Image', () => {
 
       const helperText = 'form-input-searchTerm-field-helper';
       // Wait for the validation
+      await browser.wait(
+        until.presenceOf(element(by.css('.pf-m-success[data-test-id=deploy-image-search-term]'))),
+      );
       await browser.wait(until.presenceOf(element(by.id(helperText))));
       // Confirm the results appeared
-      expect(element(by.id(helperText)).isPresent()).toBe(true);
+      expect(element(by.id(helperText)).getText()).toEqual('Validated');
```
Comment (andrewballantyne) [inline frontend/integration-tests/tests/deploy-image.scenario.ts]: cc @christianvogt

**Comment by andrewballantyne** [comment]
Got the test to pass locally, let's see it go through e2e and hopefully this is our solution.

**Comment by christianvogt** [comment]
@sahil143 you'll need to incorporate these changes into the PR that goes to 4.4 related to https://github.com/openshift/console/pull/4187

**Pull request merged**